scrc/utils/overview_creation.py

```diff
@@ -0,0 +1,47 @@
+from root import ROOT_DIR
+import os
+import sys
+import csv
+from tqdm import tqdm
+from scrc.utils.court_names import get_all_courts
+
+csv.field_size_limit(sys.maxsize)
+
+
+#   creates an overview of the size of the generated datasets and exports them in a csv file
+def create_overview(include_all=True, path="data/datasets", export_path="data", export_name="overview.csv"):
+    courts_av = os.listdir(os.path.join(ROOT_DIR, path))
+
+    #   stores the overview in a list of dicts
+    courts_data = []
+
+    #   store the number of rows of each file in a dict for each court
+    for court in tqdm(courts_av):
+        court_data = {"name": court}
+        for key in ["all", "val", "test", "train", "secret_test"]:
+            try:
```

Comment:
I guess it makes sense to create an enum for the splits and use that.

Reply:
done 👍